### PR TITLE
Allow `;` characters in merge commit messages

### PR DESCRIPTION
**Pull request by tuchk4**
Fixes https://github.com/lerna/lerna-changelog/issues/258

### CODE REVIEW

File: src/git.ts [hunk 1]
```diff
@@ -30,18 +30,31 @@ export interface CommitListItem {
...
     .map((commit: string) => {
-      const parts = commit.split(";");
-      const sha = parts[0];
-      const refName = parts[1];
-      const summary = parts[2];
-      const date = parts[3];
+      const parts = commit.match(/hash<(.+)> ref<(.*)> message<(.*)> date<(.*)>/) || [];
+
+      if (!parts) {
+        return null;
```
Comment (tuchk4) [inline src/git.ts]: Maybe we should not ignore this case. At least we can add `console.logs`

File: src/git.ts [hunk 2]
```diff
@@ -30,18 +30,31 @@ export interface CommitListItem {
...
+    ])
     .stdout.split("\n")
     .filter(Boolean)
     .map((commit: string) => {
-      const parts = commit.split(";");
-      const sha = parts[0];
-      const refName = parts[1];
-      const summary = parts[2];
-      const date = parts[3];
+      const parts = commit.match(/hash<(.+)> ref<(.*)> message<(.*)> date<(.*)>/) || [];
```
Comment (Turbo87) [inline src/git.ts]: can you extract this parsing part into a dedicated function and write a few tests for it? just to make sure that we don't regress on this 🙏

**Comment by tuchk4** [comment]
@Turbo87 what do you think?

**Comment by Turbo87** [comment]
released as v1.0.1. thanks again :)

**Pull request merged**